dev/com.ibm.websphere.appserver.features/build.gradle

```diff
@@ -72,9 +72,21 @@ task copyFeaturePiiFiles {
     Map<String,Properties> map = getFeatureMap()
     map.keySet().each { featureName ->
       Properties featureProps = map[featureName]
-      if ('public'.equals(featureProps['visibility'])) {
+      if (project.file('visibility/' + featureProps['visibility'] +'/' + featureProps['IBM-ShortName'] + '/resources').exists()) {
         copy {
-          from project.file('visibility/public/' + featureProps['IBM-ShortName'] + '/resources')
+          from project.file('visibility/' + featureProps['visibility'] +'/' + featureProps['IBM-ShortName'] + '/resources')
+          into rootProject.file('build.pii.package/nlssrc/' + featureName)
+          include 'l10n/*.properties'
+        }
+      } else if (project.file(featureProps['visibility'] + '/' + featureProps['symbolicName'] + '/resources').exists()) {
```

Comment:
This one does make sense. It should be 'visibility/' + featureProps['visibility'] + '/' + featureProps['symbolicName'] + '/resources'

This one doesn't do anything do it could be removed.

Reply:
> 
> 
> This one does make sense. It should be 'visibility/' + featureProps['visibility'] + '/' + featureProps['symbolicName'] + '/resources'
> 
> This one doesn't do anything do it could be removed.

Right, I didn't want to break existing behavior if another feature had done it that way. So I left it.

Reply:
There is no existing behavior to be left.  It only did short name before.  I suggest simplifying this like we do for the wlp/lib/features/l10n logic.

Reply:
I provided Katie some code that got rid of the need to deal with file path manipulation.